rust/operator-binary/src/discovery.rs

```diff
@@ -118,43 +79,3 @@ fn pod_hosts(hive: &HiveCluster) -> Result<impl IntoIterator<Item = (String, u16
         .into_iter()
         .map(|pod_ref| (pod_ref.fqdn(), APP_PORT)))
 }
-
-/// Lists all nodes currently hosting Pods participating in the [`Service`]
-async fn nodeport_hosts(
```

Comment:
I think this relates to a larger question of to which degree we support mixing Stackable operators with manually managed up- and downstream services (which may not be in the K8s cluster/overlay network).

Reply:
Yeah.. If this is an isolated patch then perhaps we can revert it from here and split it into a separate PR?

Reply:
Wasnt isolated, but its 5 minutes work to restore.

Reply:
:+1: